app/helpers/shifts_helper.rb

```diff
@@ -1,15 +1,15 @@
 module ShiftsHelper
 
   def shift_vote_collection_dates(polls)
-    date_options((start_date(polls)..end_date(polls)))
+    date_options((start_date(polls)..end_date(polls)), 0)
   end
 
   def shift_recount_scrutiny_dates(polls)
-    date_options(polls.map(&:ends_at).map(&:to_date).sort.inject([]) { |total, date| total << (date..date + 1.week).to_a }.flatten.uniq)
+    date_options(polls.map(&:ends_at).map(&:to_date).sort.inject([]) { |total, date| total << (date..date + 1.week).to_a }.flatten.uniq, 1)
   end
 
-  def date_options(dates)
-    dates.map { |date| [l(date, format: :long), l(date)] }
+  def date_options(dates, task_id)
```

Comment:
It bugs me (just a little bit) that a method does two things (both trimming down shift dates from task date range, and creating the date options for the select box)